netstandard/ref/System.Net.WebSockets.cs

```diff
@@ -72,8 +73,12 @@ public abstract partial class WebSocket : System.IDisposable
         public static System.ArraySegment<byte> CreateClientBuffer(int receiveBufferSize, int sendBufferSize) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
         public static System.Net.WebSockets.WebSocket CreateClientWebSocket(System.IO.Stream innerStream, string subProtocol, int receiveBufferSize, int sendBufferSize, System.TimeSpan keepAliveInterval, bool useZeroMaskingKey, System.ArraySegment<byte> internalBuffer) { throw null; }
+        public static System.Net.WebSockets.WebSocket CreateFromStream(System.IO.Stream stream, bool isServer, string subProtocol, System.TimeSpan keepAliveInterval) { throw null; }
         public static System.ArraySegment<byte> CreateServerBuffer(int receiveBufferSize) { throw null; }
         public abstract void Dispose();
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        [System.ObsoleteAttribute("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.")]
+        public static bool IsApplicationTargeting45() { throw null; }
```

Comment:
This API method makes no sense to me

Reply:
LOL... clearly @terrajobst was sleeping when he reviewed this diff as this is one of his most favorite don't do this APIs. :)

Reply:
Agreed.  Don't add this to .NET Standard.  But it was useful when we needed it during .NET Framework 4.5 development.

Reply:
Fixed